skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2007 Zachary Pincus
+# This file is part of CellTool.
+#
+# CellTool is free software; you can redistribute it and/or modify
+# it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+
+from scipy import ndimage
+import numpy
+
+def warp_images(from_points, to_points, images, output_region, interpolation_order = 1, approximate_grid=2):
+    """Define a thin-plate-spline warping transform that warps from the from_points
+    to the to_points, and then warp the given images by that transform.
+
+    References
+    ----------
+    .. [1] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+    decomposition of deformations." IEEE Transactions on pattern analysis and
+    machine intelligence 11.6 (1989): 567-585.
+
+    Parameters
+    ----------
+    from_points: (N, 2) array_like
+            Source image coordinates.
+    to_points: (N, 2) array_like
+            Target image coordinates.
+    image: ndarray
+            Image to be warped with the given warp transform.
+    output_region: (1, 4) array
+            The (xmin, ymin, xmax, ymax) region of the output
+            image that should be produced. (Note: The region is inclusive, i.e.
+            xmin <= x <= xmax)
+    interpolation_order: int, optional
+            If value is 1, then use linear interpolation else use
+            nearest-neighbor interpolation.
+    approximate_grid: int, optional
+            If approximate_grid is greater than 1, say x, then the transform is defined
+            on a grid x times smaller than the output image region. Then the
+            transform is bilinearly interpolated to the larger region. This is fairly
+            accurate for values up to 10 or so.
```

Comment:
Thanks @ana42742! The docstring is a good start. There are a few things left to conform it to the [NumPyDoc style](https://numpydoc.readthedocs.io/en/latest/format.html):

- The References section goes after the Parameter section. 
- The first sentence should a single line that sounds like an order. Followed by a blank line. 
- The Returns section is missing.

Long term we also want a example section but that should come later once we are sure ho the function will look like. :)